src/MICore/JsonLaunchOptions.cs

```diff
@@ -123,6 +124,37 @@ public abstract partial class BaseOptions
         public UnknownBreakpointHandling? UnknownBreakpointHandling { get; set; }
     }
 
+    internal class VisualizerFileConverter : JsonConverter
+    {
+        public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+        {
+            List<string> visualizerFile = new List<string>();
+            if (reader.TokenType == JsonToken.StartArray)
+            {
+                visualizerFile = serializer.Deserialize<List<string>>(reader);
+            }
+            else if (reader.TokenType == JsonToken.String)
+            {
+                visualizerFile.Add(reader.Value.ToString());
+            }
+            else
+            {
+                throw new Exception("Input is neither a List nor a String");
```

Comment:
You probably want to use a JSON serialization exception rather than `throw new Exception`. Looks like the exception would be thrown if you didn't use a JsonConverter would be `Newtonsoft.Json.JsonReaderException`, so I think I would suggest that.